.circleci/config.yml

```diff
@@ -4,8 +4,8 @@ parameters:
   workflow:
     description: Which workflow to run
     type: enum
-    enum: ['ci', 'pr', 'merged', 'daily']
-    default: 'ci'
+    enum: ['noop', 'pr', 'merged', 'daily']
```

Comment:
Should we change `'pr'` to `'normal'` as well, so it's alligned? So there isn't this implicit mapping between `ci:normal` and the `pr` workflow.

Reply:
sounds good!